programs/github

```diff
@@ -37,8 +92,21 @@ if action == 'clone' then
 
 	dest = shell.resolve(dest or repo)
 
+	local auth = nil
+
+	if flags.authed then
+		-- if no user is found auth should remain nil.
+		auth = github.Auth.get(flags.authed)
+	end
+
+	if flags.authed and not auth then
+		printError('Unknown User!')
+		print(subcommands['auth'].usage)
```

Comment:
Oh I see this case IS slightly different then other instances of us returning `printError`, in this particular case we are expecting the user to utilize a separate subcommand, and showing the user how to utilize that subcommand.  The question is still semi valid though, should we print usage for the `auth` subcommand here or just return the `printError`?